source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -1185,3 +1188,4 @@ Changelog
 .. _Client Side Operations Timeout Spec: /source/client-side-operations-timeout/client-side-operations-timeout.rst
 .. _timeoutMS: /source/client-side-operations-timeout/client-side-operations-timeout.rst#timeoutMS
 .. _t-digest algorithm: https://github.com/tdunning/t-digest
+.. _Why do we need force running prune maintenance call in Clear logic?: /source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#Why-do-we-need-force-running-prune-maintenance-call-in-Clear-logic?
```

Comment:
this link doesn't work, will fix it in the next commit

Reply:
does this link work now?